packages/cli/README.md

```diff
@@ -0,0 +1,76 @@
+# Kubernetes Secrets for Zowe CLI
+
+## Installing
+
+Use one of the following methods to install the plug-in:
+
+- Install the plug-in to Zowe CLI from an online registry or a local package.
+
+  Use this method when you want to install the plug-in quickly and start using it. With this method, you secure your user IDs and passwords manually.
```

Comment:
When you say, "Use this method," which method are you referring to, the online registry or the local package? Or do you mean to say, "Use these methods when ..."
Should we split the online registry and local package into two methods? It might be a little confusing to have two different options w/in one method.

Reply:
In this case we point to external docs for more information on both methods, I meant to say "Use these methods when ..." as a disclamer before showing the link to external docs, so I think a separation is not necessary